src/nodes/NodeConnectionManager.ts

```diff
@@ -1657,19 +1702,15 @@ class NodeConnectionManager {
     const allProms: Array<Promise<Array<void>>> = [];
     for (const targetNodeId of targetNodeIds) {
       if (!this.isSeedNode(targetNodeId)) {
+        // Ask seed nodes to signal hole punching for target
         const holePunchProms = seedNodes.map((seedNodeId) => {
           return (
-            this.sendSignalingMessage(
-              seedNodeId,
-              this.keyRing.getNodeId(),
-              targetNodeId,
-              undefined,
-              ctx,
-            )
+            this.holePunchSignalRequest(targetNodeId, seedNodeId, ctx)
               // Ignore results
               .then(
                 () => {},
-                () => {},
+                (e) =>
+                  this.logger.debug(`signal request failed with ${e.message}`),
```

Comment:
What's this?

Reply:
Nothing is being done with this exception.

Reply:
It was a debugging message from testing left in. All errors are just ignored here. It would either be a cancellation of the promise or a failure of the socket to send. In either case we don't care in this part of the code as Socket failures are handled elsewhere.

I'll remove the debugger message.